lib/Steps/Integrations/ReactNative.ts

```diff
@@ -231,9 +231,9 @@ export class ReactNative extends MobileProject {
     for (const script of buildScripts) {
       if (
         !script.shellScript.match(
-          /(packager|scripts)\/react-native-xcode\.sh\b/,
+          /\/scripts\/react-native-xcode\.sh/i,
         ) ||
-        script.shellScript.match(/sentry-cli\s+react-native[\s-]xcode/)
+        script.shellScript.match(/sentry-cli\s+react-native\s+xcode/i)
```

Comment:
Is there even a way to have more whitespace? If not, i'd also go with `includes` here.